[2025-07-31T19:41:11.691+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_pipeline_example.insert_row manual__2025-07-31T19:41:07.701638+00:00 [queued]>
[2025-07-31T19:41:11.701+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_pipeline_example.insert_row manual__2025-07-31T19:41:07.701638+00:00 [queued]>
[2025-07-31T19:41:11.702+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-07-31T19:41:11.714+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): insert_row> on 2025-07-31 19:41:07.701638+00:00
[2025-07-31T19:41:11.718+0000] {standard_task_runner.py:57} INFO - Started process 288 to run task
[2025-07-31T19:41:11.721+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'postgres_pipeline_example', 'insert_row', 'manual__2025-07-31T19:41:07.701638+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/jobs_dag.py', '--cfg-path', '/tmp/tmp_4yq4l5q']
[2025-07-31T19:41:11.724+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask insert_row
[2025-07-31T19:41:11.768+0000] {task_command.py:415} INFO - Running <TaskInstance: postgres_pipeline_example.insert_row manual__2025-07-31T19:41:07.701638+00:00 [running]> on host b94da10d01d0
[2025-07-31T19:41:11.850+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='postgres_pipeline_example' AIRFLOW_CTX_TASK_ID='insert_row' AIRFLOW_CTX_EXECUTION_DATE='2025-07-31T19:41:07.701638+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-31T19:41:07.701638+00:00'
[2025-07-31T19:41:11.852+0000] {sql.py:274} INFO - Executing: 
            INSERT INTO user_processing_data (custom_id, user_name, timestamp)
            VALUES (%(custom_id)s, %(user_name)s, %(timestamp)s);
        
[2025-07-31T19:41:11.862+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-31T19:41:11.870+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-31T19:41:11.875+0000] {sql.py:418} INFO - Running statement: 
            INSERT INTO user_processing_data (custom_id, user_name, timestamp)
            VALUES (%(custom_id)s, %(user_name)s, %(timestamp)s);
        , parameters: {'custom_id': 77431, 'user_name': "{{ ti.xcom_pull(task_ids='get_current_user') }}", 'timestamp': '2025-07-31T19:41:11.550069'}
[2025-07-31T19:41:11.876+0000] {sql.py:427} INFO - Rows affected: 1
[2025-07-31T19:41:11.887+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=postgres_pipeline_example, task_id=insert_row, execution_date=20250731T194107, start_date=20250731T194111, end_date=20250731T194111
[2025-07-31T19:41:11.942+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-07-31T19:41:11.971+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
